LayoutTests/inspector/css/node-styles-refreshed.html

```diff
@@ -55,26 +79,39 @@
         }
     });
 
-    WI.domManager.requestDocument((documentNode) => {
-        documentNode.querySelector("body", (contentNodeId) => {
-            if (contentNodeId) {
-                let domNode = WI.domManager.nodeForId(contentNodeId);
-                nodeStyles = WI.cssManager.stylesForNode(domNode);
-
-                nodeStyles.refreshIfNeeded().then(function() {
-                    suite.runTestCasesAndFinish();
-                });
-            } else {
-                InspectorTest.fail("DOM node not found.");
-                InspectorTest.completeTest();
-            }
-        });
+    suite.addTestCase({
+        name: "NodeStylesRefreshed.DisconnectedNode",
+        description: "Ensure that changing the style of a disconnected node does not crash.",
+        async test() {
+            let nodeStyles = await nodeStylesForSelector("#removable-node");
+
+            nodeStyles.singleFireEventListener(WI.DOMNodeStyles.Event.Refreshed, (event) => {
+                InspectorTest.expectFalse(event.data.significantChange, `Refreshing styles of an unchanged node should not be a significant change.`);
+            });
+
+            InspectorTest.log("-Refreshing styles before removal.")
```

Comment:
NIT: IMO having a line start with `-` makes it look like someone forgot to remove something from a diff/merge 😅